### PR TITLE
KW fixes

### PR DESCRIPTION
**Pull request by antonvor**

### CODE REVIEW

File: inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp [hunk 1]
```diff
@@ -728,6 +728,9 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
         LayerParams attrs = {node->get_friendly_name(), "DetectionOutput",
                             details::convertPrecision(node->get_output_element_type(0))};
         auto res = std::make_shared<InferenceEngine::CNNLayer>(attrs);
+        if (res == nullptr) {
```
Comment (nshchego) [inline inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp:731]: make_shared could not return nullptr. It throws exception instead.

Comment (nshchego) [reply inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp:731]: The same below.

File: inference-engine/src/mkldnn_plugin/emitters/jit_emitter.hpp [hunk 2]
```diff
@@ -28,13 +28,13 @@ class jit_emitter {
 public:
     jit_emitter(dnnl::impl::cpu::x64::jit_generator* host, dnnl::impl::cpu::x64::cpu_isa_t host_isa, const MKLDNNNode* node,
                 InferenceEngine::Precision exec_prc = InferenceEngine::Precision::FP32, emitter_in_out_map in_out_type = emitter_in_out_map::vec_to_vec)
-        : h(host), host_isa_(host_isa), exec_prc_(exec_prc), in_out_type_(in_out_type), l_table (new Xbyak::Label()) {
+        : h(host), host_isa_(host_isa), exec_prc_(exec_prc), in_out_type_(in_out_type), l_table (new Xbyak::Label()), entry_map_() {
```
Comment (nshchego) [inline inference-engine/src/mkldnn_plugin/emitters/jit_emitter.hpp:31]: I believe default ctr is called without additional call.

File: inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp (hunk 1 above)
Comment (antonvor) [reply inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp]: thanks, fixed

File: inference-engine/src/mkldnn_plugin/emitters/jit_emitter.hpp (hunk 2 above)
Comment (antonvor) [reply inference-engine/src/mkldnn_plugin/emitters/jit_emitter.hpp]: thanks, removed

**Pull request merged**